src/css/FooterTabs.css

```diff
@@ -0,0 +1,26 @@
+ul {
+	list-style-type: none;
+}
+
+li {
+	display: inline;
+	margin-left: 5px;
+}
```

Comment:
If you added `cursor: pointer;` here it would be nice indication that the user can select that item (either here or on the `.navbutton` class, I think. Not necessary, though.